components/Header/index.js

```diff
@@ -0,0 +1,78 @@
+import styled from "styled-components";
+import Image from "next/image";
+import logo from "./logo.png";
+import {Icon} from "@iconify/react";
+
+export default function Header() {
+  return (
+    <StyledHeader>
+      <Image src={logo} width="150" height="100" alt="the main logo" />
```

Comment:
The `alt` attribute is usually used as a description of what is seen in the image. Of course, the "main logo" is seen, but if the main logo is a name, for example "neuefische" then the correct way to use the `alt` attribute would be to write the content of the logo: `alt="neuefische"`. If the logo is only visual and a "random" photo, then I would suggest to leave the `alt` attribute empty. 

`alt` attributes are often used by screenreader to describe the content of an image. Imagine you can't see what is on the page and the screenreader reads: "image; the main logo". That would not be really helpful though. If it is unclear, maybe ask a coach about it 🙂 

In your case it would be: 

```suggestion
     <Image src={logo} width="150" height="100" alt="search find go" />
```